modules/ROOT/pages/installing-mobile-services.adoc

```diff
@@ -12,7 +12,9 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
 
 * link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[Ansible] (tested with version 2.4, but should work with version 2.3 or later)
 
-* link:https://www.openshift.org/download.html[Openshift client tools] version 3.9 or later
+* link:https://www.openshift.org/download.html[OpenShift client tools] version 3.9 or later
+
+WARNING: OpenShift oc executable must be located in a system location that is known to all shells (e.g. /usr/local/bin)
```

Comment:
If you add a `+` between the above step and the warning then the warning will be indented and look nicer.

Reply:
:+1: